Dockerfile.ci

```diff
@@ -1,57 +1,74 @@
-FROM golang:1.12-stretch AS go-env
+# /bin/terraform
+FROM hashicorp/terraform:0.12.8 AS terraform
 
-RUN go get -u github.com/prometheus/prometheus/cmd/promtool
+# /bin/promtool
+FROM quay.io/prometheus/prometheus:v2.12.0 AS prometheus
 
-ARG inputs_path=kapitan/inputs
-COPY $inputs_path/helm/template.go $inputs_path/helm/go.mod $inputs_path/helm/build.sh /kapitan/helm/
+# Build the Helm binding
+FROM golang:1.12-stretch AS go-builder
 
-RUN /kapitan/helm/build.sh
+RUN mkdir /kapitan
+WORKDIR /kapitan
+COPY . .
 
+RUN ./kapitan/inputs/helm/build.sh
+
+# Build final image
 FROM python:3.7-stretch
 
-RUN apt-get update && apt-get install -y \
-    build-essential git curl jq libncursesw5-dev libffi-dev zlib1g-dev libncurses5-dev libgdbm-dev libnss3-dev \
-    libssl-dev libreadline-dev libffi-dev wget libbz2-dev libsqlite3-dev bash zip lsb-release && \
-    apt-get clean && \
-    rm -rf /var/lib/apt/lists/*
+ARG CLOUD_SDK_VERSION=262.0.0
 
-ARG CLOUD_SDK_VERSION=260.0.0
 ENV CLOUD_SDK_VERSION=$CLOUD_SDK_VERSION
+ENV KAPP_URL=https://github.com/k14s/kapp/releases/download/v0.11.0/kapp-linux-amd64
+ENV KBLD_URL=https://github.com/k14s/kbld/releases/download/v0.11.0/kbld-linux-amd64
+ENV PATH="/opt/venv/bin:${PATH}"
 
-ENV PATH "$PATH:/opt/google-cloud-sdk/bin/"
+COPY --from=terraform /bin/terraform /usr/bin/terraform
+COPY --from=prometheus /bin/promtool /user/bin/promtool
+COPY --from=go-builder /kapitan /kapitan
 
-RUN export CLOUD_SDK_REPO="cloud-sdk-$(lsb_release -c -s)" && \
-    mkdir -p /opt && cd /opt && \
-    curl -O https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-sdk-${CLOUD_SDK_VERSION}-linux-x86_64.tar.gz && \
-    tar xzf google-cloud-sdk-${CLOUD_SDK_VERSION}-linux-x86_64.tar.gz && \
-    rm google-cloud-sdk-${CLOUD_SDK_VERSION}-linux-x86_64.tar.gz && \
-    gcloud config set core/disable_usage_reporting true && \
-    gcloud config set component_manager/disable_update_check true && \
-    gcloud config set metrics/environment github_docker_image && \
-    gcloud --version
+RUN apt-get -qqy update \
```

Comment:
Avoid using -qq in the commands here. Actually seeing this output in travis logs helps in debugging sometimes. :)